receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers.go

```diff
@@ -312,18 +313,77 @@ func getAge(start time.Time, end time.Time) int64 {
 func (ghs *githubScraper) getCVEs(
 	ctx context.Context,
 	client graphql.Client,
```

Comment:
```suggestion
	gClient graphql.Client,
```

Reply:
Since we have two clients here, let's specify which one is which like you did with `rClient`.

Reply:
Applied the suggestions to all instances this client is called. Pushed commit up and marking as resolved